src/Components/Projects/Projects.tsx

```diff
@@ -173,38 +194,48 @@ const Projects: React.FC = () => {
                 <th scope="col">Title</th>
                 <th scope="col">Description</th>
                 <th scope="col">Num Task</th>
-                <th scope="col">Actions</th>
+                {userRole == "Manager" ? <th scope="col">Actions</th> : ""}
               </tr>
             </thead>
             <tbody>
-              {projects?.length > 0 ? (
-                projects.map((project: any) => (
-                  <tr key={project?.id}>
-                    <td>{project?.title}</td>
-                    <td>{project?.description}</td>
-                    <td>{project?.task?.length}</td>
-                    <td>
-                      <i
-                        onClick={() => showViewModal(project?.id)}
-                        className="fa fa-eye  text-info px-2"
-                      ></i>
-                      <i
-                        onClick={() => showUpdateModal(project)}
-                        className="fa fa-pen  text-warning px-2"
-                      ></i>
-                      <i
-                        onClick={() => showDeleteModal(project.id)}
-                        className="fa fa-trash  text-danger"
-                      ></i>
-                    </td>
-                  </tr>
-                ))
+              {!isLoading ? (
+                <>
+                  {projects?.length > 0 ? (
+                    projects.map((project: any) => (
+                      <tr key={project?.id}>
+                        <td>{project?.title}</td>
+                        <td>{project?.description}</td>
+                        <td>{project?.task?.length}</td>
+                        {userRole == "Manager" ? (
+                          <td>
+                            <i
```

Comment:
We can make an icon clickable by adding onClick event to it, but this is not a good practice. and we are losing a lot of features by not using a button for that like using tap to navigate between interactive elements, and pressing space to click it, and more.

You should instead make a button and nest the icon inside it, and the button should be giving the onClick event.

Anyone who can only use the device using a keyboard will not be able to move between the pagination numbers, as they are not clickable